### ISSUE TITLE
Make GUI system work with AssetManager fonts

### ISSUE THREAD

**Issue by Dunbaratu** [KSP-KOS/KOS#1951]
(This is one of several issues that should be addressed after #1878 (GUI toolkit) gets merged into develop but before that code appears in a public realease.) It also needs to wait for #1948 (terminal fonts) to be merged into develop.

The terminal fonts PR #1948 contains a font manager system in the class ``AssetManager`` that should be usable by the GUI toolkit of PR #1878.  The GUI toolkit already has the ability to let users pick fonts but it only works on the built-in KSP fonts at the moment.  We should make it work with any OS font that ``AssetManager`` can see too.

Along with this (maybe open for discussion here) is the issue of whether or not the GUI should default to the same font as the terminal or default to the same font as the other game GUIs, or have its own separate default font setting in ``CONFIG:``.  (In any case the user should be able to explicitly pick a font - this is just for what default look we get when the user doesn't pick one).
